stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py

```diff
@@ -121,9 +121,10 @@ def _get_label_qty(self, move_line):
 
     @api.model
     def _prepare_data_from_move_line(self, move_line):
+        qty = self.env.context.get("force_quantity_line") or move_line.qty_done
```

Comment:
Do you want use move_line.qty_done if force_quantity_line is zero?
```suggestion
        qty = self.env.context.get("force_quantity_line", move_line.qty_done)
```